{"filter":false,"title":"start_bb8_move_in_a_square_service_server.launch","tooltip":"/unit_3_services/launch/start_bb8_move_in_a_square_service_server.launch","undoManager":{"mark":100,"position":100,"stack":[[{"start":{"row":2,"column":65},"end":{"row":2,"column":66},"action":"insert","lines":["e"],"id":124}],[{"start":{"row":2,"column":66},"end":{"row":2,"column":67},"action":"insert","lines":["r"],"id":125}],[{"start":{"row":2,"column":67},"end":{"row":2,"column":68},"action":"insert","lines":["v"],"id":126}],[{"start":{"row":2,"column":68},"end":{"row":2,"column":69},"action":"insert","lines":["e"],"id":127}],[{"start":{"row":2,"column":69},"end":{"row":2,"column":70},"action":"insert","lines":["r"],"id":128}],[{"start":{"row":2,"column":70},"end":{"row":2,"column":71},"action":"insert","lines":["."],"id":129}],[{"start":{"row":2,"column":71},"end":{"row":2,"column":72},"action":"insert","lines":["p"],"id":130}],[{"start":{"row":2,"column":72},"end":{"row":2,"column":73},"action":"insert","lines":["y"],"id":131}],[{"start":{"row":2,"column":73},"end":{"row":2,"column":74},"action":"insert","lines":["\""],"id":132}],[{"start":{"row":2,"column":74},"end":{"row":2,"column":75},"action":"insert","lines":[" "],"id":133}],[{"start":{"row":2,"column":75},"end":{"row":2,"column":76},"action":"insert","lines":["n"],"id":134}],[{"start":{"row":2,"column":76},"end":{"row":2,"column":77},"action":"insert","lines":["a"],"id":135}],[{"start":{"row":2,"column":77},"end":{"row":2,"column":78},"action":"insert","lines":["m"],"id":136}],[{"start":{"row":2,"column":78},"end":{"row":2,"column":79},"action":"insert","lines":["e"],"id":137}],[{"start":{"row":2,"column":79},"end":{"row":2,"column":80},"action":"insert","lines":["="],"id":138}],[{"start":{"row":2,"column":80},"end":{"row":2,"column":81},"action":"insert","lines":[":"],"id":139}],[{"start":{"row":2,"column":81},"end":{"row":2,"column":82},"action":"insert","lines":["s"],"id":140}],[{"start":{"row":2,"column":82},"end":{"row":2,"column":83},"action":"insert","lines":["e"],"id":141}],[{"start":{"row":2,"column":83},"end":{"row":2,"column":84},"action":"insert","lines":["r"],"id":142}],[{"start":{"row":2,"column":84},"end":{"row":2,"column":85},"action":"insert","lines":["v"],"id":143}],[{"start":{"row":2,"column":85},"end":{"row":2,"column":86},"action":"insert","lines":["i"],"id":144}],[{"start":{"row":2,"column":86},"end":{"row":2,"column":87},"action":"insert","lines":["c"],"id":145}],[{"start":{"row":2,"column":87},"end":{"row":2,"column":88},"action":"insert","lines":["e"],"id":146}],[{"start":{"row":2,"column":88},"end":{"row":2,"column":89},"action":"insert","lines":["_"],"id":147}],[{"start":{"row":2,"column":89},"end":{"row":2,"column":90},"action":"insert","lines":["m"],"id":148}],[{"start":{"row":2,"column":90},"end":{"row":2,"column":91},"action":"insert","lines":["o"],"id":149}],[{"start":{"row":2,"column":91},"end":{"row":2,"column":92},"action":"insert","lines":["v"],"id":150}],[{"start":{"row":2,"column":92},"end":{"row":2,"column":93},"action":"insert","lines":["e"],"id":151}],[{"start":{"row":2,"column":93},"end":{"row":2,"column":94},"action":"insert","lines":["_"],"id":152}],[{"start":{"row":2,"column":80},"end":{"row":2,"column":81},"action":"remove","lines":[":"],"id":153}],[{"start":{"row":2,"column":80},"end":{"row":2,"column":81},"action":"insert","lines":["\""],"id":154}],[{"start":{"row":2,"column":94},"end":{"row":2,"column":95},"action":"insert","lines":["b"],"id":155}],[{"start":{"row":2,"column":95},"end":{"row":2,"column":96},"action":"insert","lines":["b"],"id":156}],[{"start":{"row":2,"column":96},"end":{"row":2,"column":97},"action":"insert","lines":["8"],"id":157}],[{"start":{"row":2,"column":97},"end":{"row":2,"column":98},"action":"insert","lines":["_"],"id":158}],[{"start":{"row":2,"column":98},"end":{"row":2,"column":99},"action":"insert","lines":["i"],"id":159}],[{"start":{"row":2,"column":99},"end":{"row":2,"column":100},"action":"insert","lines":["n"],"id":160}],[{"start":{"row":2,"column":100},"end":{"row":2,"column":101},"action":"insert","lines":["_"],"id":161}],[{"start":{"row":2,"column":101},"end":{"row":2,"column":102},"action":"insert","lines":["s"],"id":162}],[{"start":{"row":2,"column":102},"end":{"row":2,"column":103},"action":"insert","lines":["q"],"id":163}],[{"start":{"row":2,"column":103},"end":{"row":2,"column":104},"action":"insert","lines":["u"],"id":164}],[{"start":{"row":2,"column":104},"end":{"row":2,"column":105},"action":"insert","lines":["a"],"id":165}],[{"start":{"row":2,"column":105},"end":{"row":2,"column":106},"action":"insert","lines":["r"],"id":166}],[{"start":{"row":2,"column":106},"end":{"row":2,"column":107},"action":"insert","lines":["e"],"id":167}],[{"start":{"row":2,"column":107},"end":{"row":2,"column":108},"action":"insert","lines":["_"],"id":168}],[{"start":{"row":2,"column":108},"end":{"row":2,"column":109},"action":"insert","lines":["s"],"id":169}],[{"start":{"row":2,"column":109},"end":{"row":2,"column":110},"action":"insert","lines":["e"],"id":170}],[{"start":{"row":2,"column":110},"end":{"row":2,"column":111},"action":"insert","lines":["r"],"id":171}],[{"start":{"row":2,"column":111},"end":{"row":2,"column":112},"action":"insert","lines":["v"],"id":172}],[{"start":{"row":2,"column":112},"end":{"row":2,"column":113},"action":"insert","lines":["e"],"id":173}],[{"start":{"row":2,"column":113},"end":{"row":2,"column":114},"action":"insert","lines":["r"],"id":174}],[{"start":{"row":2,"column":114},"end":{"row":2,"column":115},"action":"insert","lines":["\""],"id":175}],[{"start":{"row":2,"column":115},"end":{"row":2,"column":116},"action":"insert","lines":[" "],"id":176}],[{"start":{"row":2,"column":116},"end":{"row":2,"column":117},"action":"insert","lines":["o"],"id":177}],[{"start":{"row":2,"column":117},"end":{"row":2,"column":118},"action":"insert","lines":["u"],"id":178}],[{"start":{"row":2,"column":118},"end":{"row":2,"column":119},"action":"insert","lines":["t"],"id":179}],[{"start":{"row":2,"column":119},"end":{"row":2,"column":120},"action":"insert","lines":["p"],"id":180}],[{"start":{"row":2,"column":120},"end":{"row":2,"column":121},"action":"insert","lines":["u"],"id":181}],[{"start":{"row":2,"column":121},"end":{"row":2,"column":122},"action":"insert","lines":["t"],"id":182}],[{"start":{"row":2,"column":122},"end":{"row":2,"column":123},"action":"insert","lines":["="],"id":183}],[{"start":{"row":2,"column":123},"end":{"row":2,"column":124},"action":"insert","lines":["\""],"id":184}],[{"start":{"row":2,"column":124},"end":{"row":2,"column":125},"action":"insert","lines":["s"],"id":185}],[{"start":{"row":2,"column":125},"end":{"row":2,"column":126},"action":"insert","lines":["c"],"id":186}],[{"start":{"row":2,"column":126},"end":{"row":2,"column":127},"action":"insert","lines":["r"],"id":187}],[{"start":{"row":2,"column":127},"end":{"row":2,"column":128},"action":"insert","lines":["e"],"id":188}],[{"start":{"row":2,"column":128},"end":{"row":2,"column":129},"action":"insert","lines":["e"],"id":189}],[{"start":{"row":2,"column":129},"end":{"row":2,"column":130},"action":"insert","lines":["n"],"id":190}],[{"start":{"row":2,"column":130},"end":{"row":2,"column":131},"action":"insert","lines":["\""],"id":191}],[{"start":{"row":2,"column":131},"end":{"row":2,"column":132},"action":"insert","lines":[" "],"id":192}],[{"start":{"row":2,"column":132},"end":{"row":2,"column":133},"action":"insert","lines":["/"],"id":193}],[{"start":{"row":2,"column":133},"end":{"row":2,"column":134},"action":"insert","lines":[">"],"id":194}],[{"start":{"row":2,"column":134},"end":{"row":3,"column":0},"action":"insert","lines":["",""],"id":195},{"start":{"row":3,"column":0},"end":{"row":3,"column":4},"action":"insert","lines":["    "]}],[{"start":{"row":3,"column":0},"end":{"row":3,"column":4},"action":"remove","lines":["    "],"id":196}],[{"start":{"row":2,"column":134},"end":{"row":3,"column":0},"action":"remove","lines":["",""],"id":197}],[{"start":{"row":2,"column":134},"end":{"row":3,"column":0},"action":"insert","lines":["",""],"id":198},{"start":{"row":3,"column":0},"end":{"row":3,"column":4},"action":"insert","lines":["    "]}],[{"start":{"row":3,"column":0},"end":{"row":3,"column":4},"action":"remove","lines":["    "],"id":199}],[{"start":{"row":3,"column":0},"end":{"row":3,"column":1},"action":"insert","lines":["<"],"id":200}],[{"start":{"row":3,"column":1},"end":{"row":3,"column":2},"action":"insert","lines":["/"],"id":201}],[{"start":{"row":3,"column":2},"end":{"row":3,"column":3},"action":"insert","lines":["l"],"id":202}],[{"start":{"row":3,"column":3},"end":{"row":3,"column":4},"action":"insert","lines":["a"],"id":203}],[{"start":{"row":3,"column":4},"end":{"row":3,"column":5},"action":"insert","lines":["u"],"id":204}],[{"start":{"row":3,"column":5},"end":{"row":3,"column":6},"action":"insert","lines":["n"],"id":205}],[{"start":{"row":3,"column":6},"end":{"row":3,"column":7},"action":"insert","lines":["c"],"id":206}],[{"start":{"row":3,"column":7},"end":{"row":3,"column":8},"action":"insert","lines":["h"],"id":207}],[{"start":{"row":3,"column":8},"end":{"row":3,"column":9},"action":"insert","lines":[">"],"id":208}],[{"start":{"row":2,"column":14},"end":{"row":2,"column":15},"action":"insert","lines":["\""],"id":209}],[{"start":{"row":2,"column":38},"end":{"row":2,"column":71},"action":"remove","lines":["bb8_move_in_square_service_server"],"id":210},{"start":{"row":2,"column":38},"end":{"row":2,"column":74},"action":"insert","lines":["bb8_move_in_square_service_server.py"]}],[{"start":{"row":2,"column":74},"end":{"row":2,"column":75},"action":"remove","lines":["."],"id":211}],[{"start":{"row":2,"column":74},"end":{"row":2,"column":75},"action":"remove","lines":["p"],"id":212}],[{"start":{"row":2,"column":74},"end":{"row":2,"column":75},"action":"remove","lines":["y"],"id":213}],[{"start":{"row":2,"column":15},"end":{"row":2,"column":30},"action":"remove","lines":["unit_3_services"],"id":214},{"start":{"row":2,"column":15},"end":{"row":2,"column":30},"action":"insert","lines":["unit_3_services"]}],[{"start":{"row":2,"column":133},"end":{"row":2,"column":134},"action":"remove","lines":["/"],"id":215}],[{"start":{"row":2,"column":132},"end":{"row":2,"column":133},"action":"remove","lines":[" "],"id":216}],[{"start":{"row":2,"column":133},"end":{"row":3,"column":0},"action":"insert","lines":["",""],"id":217},{"start":{"row":3,"column":0},"end":{"row":3,"column":4},"action":"insert","lines":["    "]}],[{"start":{"row":3,"column":4},"end":{"row":3,"column":5},"action":"insert","lines":["<"],"id":218}],[{"start":{"row":3,"column":5},"end":{"row":3,"column":6},"action":"insert","lines":["/"],"id":219}],[{"start":{"row":3,"column":6},"end":{"row":3,"column":7},"action":"insert","lines":["n"],"id":220}],[{"start":{"row":3,"column":7},"end":{"row":3,"column":8},"action":"insert","lines":["o"],"id":221}],[{"start":{"row":3,"column":8},"end":{"row":3,"column":9},"action":"insert","lines":["d"],"id":222}],[{"start":{"row":3,"column":9},"end":{"row":3,"column":10},"action":"insert","lines":["e"],"id":223}],[{"start":{"row":3,"column":10},"end":{"row":3,"column":11},"action":"insert","lines":[">"],"id":224}]]},"ace":{"folds":[],"scrolltop":0,"scrollleft":573.5999755859375,"selection":{"start":{"row":3,"column":11},"end":{"row":3,"column":11},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"timestamp":1538113303172,"hash":"df2e489a08eea5667c5f0e316dd3f17bde827f08"}